stdlib/camlinternalFormat.ml

```diff
@@ -537,6 +544,16 @@ let bprint_fmt buf fmt =
     | Theta rest ->
       buffer_add_char buf '%'; bprint_ignored_flag buf ign_flag;
       buffer_add_char buf 't'; fmtiter rest false;
+    | Custom (arity, _, rest) ->
+      let n = int_of_custom_arity arity in
+      if n > 0 then begin
+        buffer_add_char buf '%'; bprint_ignored_flag buf ign_flag;
+        buffer_add_char buf '?';
+        for _i = 1 to n - 1 do
+          buffer_add_string buf "%?"
```

Comment:
This is the only part of the patch that might be wrong. Shouldn't we use `bprint_ignored_flag` on all `n` parameters?